streams/src/main/java/org/apache/kafka/streams/state/internals/InMemorySessionStore.java

```diff
@@ -202,25 +205,43 @@ public void remove(final Windowed<Bytes> sessionKey) {
 
     @Override
     public byte[] fetchSession(final Bytes key,
-                               final long earliestSessionEndTime,
-                               final long latestSessionStartTime) {
+                               final long sessionStartTime,
```

Comment:
This is a minor fix on the param names: the old ones are simply wrong and misleading.